internal/pkg/cli/task_run_test.go

```diff
@@ -1062,7 +1062,7 @@ func TestTaskRunOpts_Execute(t *testing.T) {
 				m.store.EXPECT().GetApplication("my-app").Return(&config.Application{Name: "my-app"}, nil).Times(2)
 				m.deployer.EXPECT().GetTaskStack(inGroupName).Return(&info, nil)
 				key := "manual/env-files/magic.env/4963d64294508aa3fa103ccac5ad1537944c577d469608ddccad09b6f79b6406.env"
-				arn := "arn:aws:s3:::bigbucket/" + key
+				arn := "https://bigbucket.s3-us-west-2.amazonaws.com/" + key
```

Comment:
Is `Upload` expected to return an URL, instead of ARN now? If so, can we also rename the variable to `bucketURL`?

Reply:
it always expects a url...it is a bad test in the past. i'll fix the var name too